src/styles/index.css

```diff
@@ -719,3 +719,9 @@ ol.mdx-illustration-block {
     width: 100%;
   }
 }
+
+.exit {
```

Comment:
Nice catch. There's nothing to click because it switches to the Thank you after it submits when this class is applied, but the cursor still switches to text select so I'll add this.

Reply:
https://github.com/reactjs/react.dev/pull/6541